backend/internal/models/user.go

```diff
@@ -13,3 +13,14 @@ type User struct {
 	CreatedAt      time.Time `json:"created_at"`
 	UpdatedAt      time.Time `json:"updated_at"`
 }
+
+type Notification struct {
+	ID               int       `json:"id"`
+	Thumbnail        string    `json:"thumbnail"` // Url of the notification thumbnail
+	CreatedAt        time.Time `json:"created_at"`
+	Type             string    `json:"type"`        // Type of the notifcation
+	RecieverID       string    `json:"reciever_id"` // Reciever of the notification
```

Comment:
apologies it's the spelling bee nerd in me

```suggestion
	ReceiverID       string    `json:"receiver_id"` // Receiver of the notification
```

Reply:
oops

Reply:
i before e except after c